### ISSUE TITLE
A lot of UI bugs (1.0.0 | windows | dark theme)

### ISSUE THREAD

**Issue by GitStorageOne** [pgmodeler/pgmodeler#1738]
**Bug description**
With **dark theme**, in some places, text color is black and barely readable.  
Please note, that I tried to delete "C:\Users\Username\AppData\Local\pgmodeler-1.0" folder.

**How to reproduce**
- Open pgModeler.
- Open settings 
- Text of every checkbox and radio button is black.
- Same applies not only for settings form, but for entire application.
---
- Open About window
- Text of hyperlink is too dark and barely readable (I assume that same issue applies to all hyperlinks)
- Text of version number also too dark.
---
- Disabled tool buttons: "Zoom in" / "Zoom out" / "Normal zoom" / "Toggle of magnifier" - does not look well with dark theme.
---
- Open demo model.
- Press "Design" button on the left panel.
- Schema name should be white instead of black.
- Open "model objects" view - all list box items are with black text, not readable.
- Same issue with lists applies to entire application (for example, "access to plugin settings" - "quick edit"; database list, attribute list).
---
- Open demo model.
- Press "Manage" button on the left panel.
- Open "Source code" viewer and attributes, select any database to fill up attributes.
- Now we see a lot of unnecessary borders for panels, which makes the user interface look very overloaded. Almost all gray borders might be removed and as for resize panel - you can change color on mouse hover instead.
---
- Open demo model.
- Press "Manage" button on the left panel.
- Select any database.
- Click on query editor field.
- Now, selected list item becomes white
---
- Open demo model.
- Press "Manage" button on the left panel.
- Select any database.
- Expand / collapse list - background of list will flick. 
---
- Open settings
- There is "configurations directory" input field with "Open in file manager" button - when I press it, nothing happens.

**Expected behavior**
Clean UI for version 1.0.0 that was tested on all supported platforms.

**Screenshots**
![image](https://user-images.githubusercontent.com/3978064/216122335-e1ffd614-e585-408b-8572-c3bb673bdd1e.png)
![image](https://user-images.githubusercontent.com/3978064/216122504-0c208070-1722-4896-83f8-935194e0dc96.png)
![image](https://user-images.githubusercontent.com/3978064/216122046-a47f8171-2b50-4c26-991e-d6375f0b415e.png)
![image](https://user-images.githubusercontent.com/3978064/216122609-cb936c07-1177-438c-8ae6-8ba039a611da.png)
![image](https://user-images.githubusercontent.com/3978064/216122798-511400b8-1eac-49be-9a64-76e606944d28.png)
![image](https://user-images.githubusercontent.com/3978064/216123071-8dbc346d-b213-4e12-a578-e9c26a610842.png)
![image](https://user-images.githubusercontent.com/3978064/216123746-4f4ebb93-f291-4738-9cd5-d59205e0f63b.png)
![image](https://user-images.githubusercontent.com/3978064/216124804-d49f03da-9357-4c18-93b5-8d4fa3f2aef6.png)

**Info about your desktop**
 - OS: Windows 10
 - Version: 10.0.19044.2486
 - Window manager: Explorer
 - pgModeler version: 1.0.0
 - Qt version: 6.4.1

**Comment by rkhaotix** [comment]
Oh man, Windows always surprises me! :/

How about the light theme? It works fine?

**Comment by GitStorageOne** [comment]
With white theme, there are only few issues.  

- "Open in file manager" button for obvious reasons does not work with light theme too.
- Unnecessary borders for panels are also applies for white theme.
- Icons is not sharp because of black shadow (especially icons with black sides, like "new model"),  
![image](https://user-images.githubusercontent.com/3978064/216152110-543a1d82-8126-4c36-a24b-a7b2c6835cf2.png)  
![image](https://user-images.githubusercontent.com/3978064/216157218-474ba533-7a09-4f32-862c-17dd0427f622.png)


- Also black text with black shadow does not look well, you want to use light shadow with black text.  
Compare how looks text under button and hovered button, text inside hovered button is much more sharp and nice.  
![image](https://user-images.githubusercontent.com/3978064/216152287-e57a6aa1-4af2-4fd9-b5f7-120d5a002b5e.png)

But, with current background color, for black texts, I recommend shadow `text-shadow: #fff 0px 1px 0px;`:
![image](https://user-images.githubusercontent.com/3978064/216155516-e097b7d0-e578-4c4e-980e-a029780e7169.png)

For blue background and black text: `text-shadow: #8ad8ff 0px 1px 0px;`  
![image](https://user-images.githubusercontent.com/3978064/216156825-a2372543-987f-4c87-a927-c1a2f0301bc9.png)

**Comment by rkhaotix** [comment]
Ok. I found the culprit. I reverted the last commit and I'll upload the new binaries in a few minutes. For the suggestions on the light theme, I'll apply them on pgModeler 1.0.1. Anyway, thanks for reporting.

**Comment by GitStorageOne** [comment]
> Ok. I found the culprit. I reverted the last commit and I'll upload the new binaries in a few minutes. For the suggestions on the light theme, I'll apply them on pgModeler 1.0.1. Anyway, thanks for reporting.

I numerated problems with black theme for convenient. Hope you will fix them all within few next patches.

**Comment by GitStorageOne** [comment]
First post updated to reflect fixes of 1.0.2.  
2, 3, 5, 8, 10 - still there.

**Comment by GitStorageOne** [comment]
@rkhaotix  
Are there any plans to fix the remaining issues?  
Or at least some of them?

**Comment by rkhaotix** [comment]
Yes, I do. I'm working on some urgent problems but I'll return to this issue. Don't worry. :)

**Comment by GitStorageOne** [comment]
First post updated to reflect fixes of 1.1.0
